bears/general/LineLengthBear.py

```diff
@@ -30,7 +47,30 @@ def run(self,
         :param indent_size:         Number of spaces per indentation level.
         :param ignore_length_regex: Lines matching each of the regular
                                     expressions in this list will be ignored.
+        :param language:            Programming language of the source code.
         '''
+        if language:
+            try:
+                language = Language[language]
+            except UnknownLanguageError:
+                logging.warn(
+                    language + ' is an invalid language name '
+                    'or is not recognized by coala. Language specific '
+                    'line length limitation can not be applied.'
+                )
+                language = Language['Unknown']
+            else:
+                if 'max_line_length' in language.attributes:
+                    max_line_length = language.max_line_length
```

Comment:
Ideally coalib sets `max_line_length` automatically in *coalib* using aspects + language.

A project for another day ... ;-)